frontend/elements/src/entry/home/TOSORT/column-list.ts

```diff
@@ -32,13 +44,15 @@ export class _ extends LitElement {
 
   @property()
```

Comment:
colors and sizes should be an enum or union type

Reply:
Fixed